specs/hotel_spec.rb

```diff
@@ -0,0 +1,357 @@
+require_relative 'spec_helper'
+
+describe Hotel::Hotel do
+  describe "a hotel instance can be created" do
+    it "can be created" do
+      marriott = Hotel::Hotel.new
+      marriott.must_be_instance_of Hotel::Hotel
+    end
+
+    it "creates 20 rooms upon initializing stored in an Array" do
+      marriott = Hotel::Hotel.new
+      marriott.rooms.length.must_equal 20
+      marriott.rooms.must_be_kind_of Array
+    end
+
+    it "creates 20 rooms that are room objects" do
+      marriott = Hotel::Hotel.new
+      marriott.rooms.each do |room|
+        room.must_be_instance_of Hotel::Room
+      end
+    end
+
+    it "creates 20 hotels that can identify their room numbers" do
```

Comment:
This is probably too specific as to the implementation of the rooms, like that they are in order.  Maybe just test that `rooms` must include a room with each id.